dags/export_dag.py

```diff
@@ -98,7 +98,7 @@ def get_boolean_env_variable(env_variable_name, default=True):
     setup_command + ' && ' + \
     '$PYTHON3 export_traces.py -b $EXPORT_BATCH_SIZE -w $EXPORT_MAX_WORKERS -s $START_BLOCK -e $END_BLOCK ' \
     '-p $WEB3_PROVIDER_URI_ARCHIVAL -o traces.csv ' \
-    '--genesis-traces --daofork-traces && ' \
+    '$EXPORT_DAOFORK_TRACES $EXPORT_GENESIS_TRACES && ' \
```

Comment:
One last thing. Seems here you missed $EXPORT_TRACES_EXTRA_PARAMS?

Reply:
In fact, I just read your previous comment more carefully 
> It allows passing the parameters for traces as ENV VARs, specifically --genesis-traces & --daofork-traces. That is the default. It allows passing instead --no-daofork-traces in the ENV VAR as an option in $EXPORT_DAOFORK_TRACES' & $EXPORT_GENESIS_TRACESfor--genesis-traces`.

I forgot that one can use --no-daofork-traces and --no-genesis-traces. So your previous solution is more straightforward and better. So I will merge the PR and revert to your previous solution for special traces.